module/edge_logic.lua

```diff
@@ -16,9 +16,9 @@ local belt_type_to_loader_type = {
 }
 
 
-local function vec2_sadd(a, s)
-	return {a[1] + s, a[2] + s}
-end
+-- local function vec2_sadd(a, s)
+-- 	return {a[1] + s, a[2] + s}
+-- end
```

Comment:
Do we need to uncomment unused functions? They are still useful when changing the vector math for the edge logic.